Requirements/ViewModels/RequirementBrowser/IRequirementBrowserDisplaySettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ------------------------------------------------------------------------------------------------
+// <copyright file="IRequirementBrowserDisplaySettings.cs" company="RHEA System S.A.">
+//   Copyright (c) 2015-2019 RHEA System S.A.
+// </copyright>
+// -----------------------------------------------------------------------------------------------
+
+namespace CDP4Requirements.ViewModels.RequirementBrowser
+{
+    /// <summary>
+    /// Interface that is used for hiding/showing certain rows in the RequirementBrowser
+    /// </summary>
+    public interface IRequirementBrowserDisplaySettings
+    {
+        bool IsSimpleParameterTypeDisplayed { get; set; }
```

Comment:
Should this property not be called **IsSimpleParameterValuesDisplayed**

can we add some docs here please, something like:
Gets or sets a value indicating whether the SimpleParameterValues are displayed